### ISSUE TITLE
useViewModel does not work correctly with '*' for the aggregateIds

### ISSUE THREAD

**Issue by oliversturm** [reimagined/resolve#1596]
I'm using a `*` to query a view model:

```js
useViewModel('products', '*', setProducts);
```

There's an error shown on the console, and updates are not working.

```
Error while handling message from websocket: Event filter field "aggregateIds" should be array of strings or be null Error: Event filter field "aggregateIds" should be array of strings or be null
    at Object.validateEventFilter (/home/user/git/resolve-tests/resolve-intro-step1/node_modules/resolve-eventstore-base/src/validate-event-filter.js:58:11)
    at args (/home/user/git/resolve-tests/resolve-intro-step1/node_modules/resolve-eventstore-base/src/wrap-event-filter.js:2:8)
    at wrappedMethod (/home/user/git/resolve-tests/resolve-intro-step1/node_modules/resolve-eventstore-base/src/wrap-method.js:10:48)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at WebSocket.<anonymous> (/home/user/git/resolve-tests/resolve-intro-step1/dist/common/local-entry/node_modules/resolve-runtime/lib/local/init-websockets.js:97:12)
```

I used that syntax because Intellisense told me that it's supposed to be this way - and the data which is returned is also correct. But the error is generated during the query, and I suspect that the subscription isn't working correctly - unless I had to do something else for that. I don't see updates to the view model when I execute commands.

If I pass `['<some guid>']`, everything works correctly (but of course I only see one row of data). If I pass `null` (since the error message says so), I see nothing (but there's no error). If I pass `['*']`, everything is the same as for `'*'` - I guess the `*` handling is not properly implemented on some level.

**Comment by timbset** [comment]
It's a bug. We'll check what is the problem.

**Comment by timbset** [comment]
Fixed in 0.25.13
